src/Service.Tests/dab-config.MsSql.json

```diff
@@ -1,868 +1,1832 @@
 {
-  "$schema": "../../schemas/dab.draft.schema.json",
+  "$schema": "https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json",
   "data-source": {
     "database-type": "mssql",
+    "connection-string": "Server=tcp:127.0.0.1,1433;Persist Security Info=False;User ID=sa;Password=REPLACEME;MultipleActiveResultSets=False;Connection Timeout=5;",
     "options": {
       "set-session-context": true
-    },
-    "connection-string": "Server=tcp:127.0.0.1,1433;Persist Security Info=False;User ID=sa;Password=REPLACEME;MultipleActiveResultSets=False;Connection Timeout=5;"
+    }
   },
   "runtime": {
     "rest": {
       "enabled": true,
       "path": "/api"
     },
     "graphql": {
-      "allow-introspection": true,
       "enabled": true,
-      "path": "/graphql"
+      "path": "/graphql",
+      "allow-introspection": true
     },
     "host": {
-      "mode": "development",
       "cors": {
-        "origins": [
-          "http://localhost:5000"
-        ],
+        "origins": ["http://localhost:5000"],
         "allow-credentials": false
       },
       "authentication": {
-        "provider": "StaticWebApps"
-      }
+        "provider": "StaticWebApps",
+        "jwt": {
+          "audience": null,
+          "issuer": null
+        }
+      },
+      "mode": "development"
     }
   },
   "entities": {
     "Publisher": {
-      "source": "publishers",
+      "source": {
+        "object": "publishers",
+        "type": "table",
+        "parameters": null,
+        "key-fields": null
+      },
+      "graphql": {
+        "enabled": true,
+        "operation": null,
+        "type": {
+          "singular": "Publisher",
+          "plural": "Publishers"
+        }
+      },
+      "rest": {
+        "enabled": true,
+        "path": null,
+        "methods": []
+      },
```

Comment:
You don't have to provide all parameters, `"rest": true"` is still valid, it's just when you do future _serialization_ it will always expand to reduce the complexity of the `JsonConverter.Write` method from having to know about short-hand properties.

There's probably cases where truly optional JSON properties, like `parameters` or `key-fields` could be removed from serialization, but that'd require a custom converter + logic in the `Write` method.